map_annotator/CMakeLists.txt

```diff
@@ -0,0 +1,178 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(map_annotator)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS 
+  rospy
+  map_annotator_msgs
```

Comment:
Any dependencies you declare here also need to be in the package.xml or the package won't build.